plugins/georeplication/api/resp.go

```diff
@@ -54,3 +54,22 @@ type GeorepSession struct {
 	Workers    []GeorepWorker    `json:"workers"`
 	Options    map[string]string `json:"options"`
 }
+
+// NewGeorepSession creates new instance of GeorepSession
+func NewGeorepSession(mastervolid uuid.UUID, slavevolid uuid.UUID, req GeorepCreateReq) *GeorepSession {
```

Comment:
This shouldn't be part of `api` package though which is supposed to contain only API requests and responses. In future, I hope we will be able to generate API doc from the package or generate package code from the API spec.

You can place this function near the handler. And if it's only invoked there, you can as well choose not export it.